microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md

```diff
@@ -30,6 +30,8 @@ To learn about how to publish source maps, see [Securely debug original code by
 
 To obtain source maps from the Azure Artifacts symbol server, DevTools uses the fetch protocol to talk to Azure DevOps, which requires a valid Personal Access Token (PAT).
 
+You need to generate a PAT even if you already generated one to publish source maps to the symbol server in [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](publish-source-maps-to-azure.md).
+
 To generate a PAT in Azure DevOps:
 
```

Comment:
Should the image after step 2 and the text in step 3 match the changes to from the Publish version in the other changed file? The text and image don't exactly match here either ... image is "devtool source maps" vs step 3 "DevTools source maps" so we can fix this wording or replace both with the new version.

Reply:
@captainbrosset @vchapel 

discussion from other PR: https://github.com/MicrosoftDocs/edge-developer/pull/1766#discussion_r818468903
I imagine giving some specific advised, suggested names such as this breakout:

Publish page's image: "DevTools source maps - Admin PAT" or "Admin PAT for DevTools source maps"
Publish page's text: "DevTools source maps - Admin PAT" or "Admin PAT for DevTools source maps"

Consume page's image: "DevTools source maps - User PAT" or "User PAT for DevTools source maps"
Consume page's text: "DevTools source maps - User PAT" or "User PAT for DevTools source maps"

Reply:
@vchapel you're right, the text on the Consume page didn't exactly match the image. The text was "DevTools source maps" while the image showed "devtool source maps". I fixed that.

> Should the image after step 2 and the text in step 3 match the changes to from the Publish version in the other changed file?

I'm not following you here. The PAT created in the Publish page has nothing to do with the PAT on the Consume page. Those are 2 different things.

@mikehoffms the 2 pages have the advices now:

* Publish page has: `enter a name for the PAT, such as "publish source maps"`
* Consume page has: `enter a name for the PAT, such as "devtool source maps"`

And both images show these strings too.